libs/core/src/lib/popover/popover.component.ts

```diff
@@ -176,9 +180,12 @@ export class PopoverComponent extends BasePopoverClass implements AfterViewInit,
     /** @hidden */
     @HostListener('keydown', ['$event'])
     onKeyDown(event: KeyboardEvent): void {
+        const activeElement = document.activeElement;
         if (
-            this.popoverControl.elRef.nativeElement.children[0] === document.activeElement &&
-            document.activeElement.tagName !== 'INPUT' && document.activeElement.tagName !== 'TEXTAREA'
+            this.popoverControl.elRef.nativeElement.children[0] === activeElement &&
+            activeElement.tagName !== 'INPUT' &&
```

Comment:
May be it's better to extract these `INPUT`, `TEXTAREA` to constants?

Reply:
I think we can count on these tagNames never changing so there is no added benefit of moving these to constants